crates/amaru-kernel/src/lib.rs

```diff
@@ -152,6 +152,12 @@ impl Point {
             Point::Specific(slot, _) => *slot,
         }
     }
+    pub fn pretty_point(&self) -> String {
+        match self {
+            Point::Origin => "origin".to_string(),
+            Point::Specific(_slot, hash) => String::from_utf8(hex::encode(hash).into()).unwrap(),
```

Comment:
`unwrap` shouldn't be used

Reply:
Definitely :) There is one already, your commit doesn't pass the [sanity check](https://github.com/pragma-org/amaru/actions/runs/14036257998/job/39294877094)